### PR TITLE
Feature/scrum 3762

### PR DESCRIPTION
**Pull request by tlm**

### CODE REVIEW

File: src/main/java/net/apnic/whowas/error/controller/ErrorController.java [hunk 1]
```diff
@@ -32,8 +53,25 @@ public void handle(Exception ex)
      * @param ex MalformedRequestException to handle
      */
     @ExceptionHandler(MalformedRequestException.class)
-    @ResponseStatus(HttpStatus.BAD_REQUEST)
-    public void handle(MalformedRequestException ex)
+    public ResponseEntity<TopLevelObject> handle(HttpServletRequest request,
+        MalformedRequestException ex)
+    {
+        LOGGER.debug("Malformed request recieved");
```
Comment (rlcintra) [inline src/main/java/net/apnic/whowas/error/controller/ErrorController.java]: Typo on "received". (small)

**Review by rlcintra — APPROVED** [review]

**Review by rlcintra — APPROVED** [review]

File: src/main/java/net/apnic/whowas/App.java [hunk 2]
```diff
@@ -56,7 +57,16 @@
...
+        SpringApplication app = new SpringApplication(App.class);
+        Properties defaultProps = new Properties();
+
+        defaultProps.setProperty(
+                "spring.mvc.throw-exception-if-no-handler-found", "true");
+        defaultProps.setProperty("spring.resources.add-mappings", "false");
+        defaultProps.setProperty("spring.mvc.favicon.enabled", "false");
+        defaultProps.setProperty("management.add-application-context-header", "false");
+        app.setDefaultProperties(defaultProps);
+        app.run(args);
```
Comment (GKTheOne) [inline src/main/java/net/apnic/whowas/App.java:69]: {comment} Why not use baked in properties to set these?

**Comment by apnicjim** [comment]
Are there any tests that show what features were implemented?

File: src/main/java/net/apnic/whowas/rdap/config/RDAPConfiguration.java [hunk 3]
```diff
@@ -1,19 +1,169 @@
...
+        configNotices = null;
+    }
+
+    /**
+     * Bean for a list of default RDAP notices.
+     */
+    @Bean
+    public List<Notice> defaultNotices()
+    {
+        return defaultNotices;
```
Comment (GKTheOne) [inline src/main/java/net/apnic/whowas/rdap/config/RDAPConfiguration.java]: {comment} Throw the post-construct / immutable magic here?

**Review by GKTheOne — APPROVED** [review]

**Pull request merged**